src/main/resources/interface-specification.yml

```diff
@@ -741,6 +789,14 @@ components:
       schema:
         type: string
   schemas:
+    RejectPurposeVersionPayload:
+      type: object
+      description: models the reject payload for this purpose version.
+      properties:
+        rejectionReason:
+          type: string
```

Comment:
[field constraints](https://github.com/pagopa/interop-be-backend-for-frontend/pull/269/files#diff-17a3d713b849787a0ed57552f2b367a47eedf4cef81a6ba025d0060ac107d213R10295-R10296) should go in the process